mozilla-build/setup

```diff
@@ -40,6 +40,6 @@ git pull
 popd
 
 echo Generating blame information
-CINNABAR=1 python $MOZSEARCH_PATH/blame/transform-repo.py $GIT_ROOT $BLAME_ROOT
```

Comment:
You can't miss the `CINNABAR=1` unless I'm missing something, right?

Reply:
The last patch in https://github.com/mozsearch/mozsearch/pull/311/commits made CINNABAR=1 the default. It's still possible to set CINNABAR=0 to explicitly disable cinnabar usage, but that should only be needed if cinnabar is not installed, otherwise we'll just get `None` for hg revs and skip the hg info.

Reply:
Ah woops, forgot about it. Looks good then, carry on :)